typescript/sdk/src/deploy/verify/utils.ts

```diff
@@ -28,7 +28,7 @@ export function getContractVerificationInput(
   isProxy: boolean = name.endsWith('Proxy'),
 ): ContractVerificationInput {
   return {
-    name,
+    name: name.charAt(0).toUpperCase() + name.slice(1),
```

Comment:
did you have trouble with verifying and this was the fix? thanks for the patch

Reply:
yessir